File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.swift
import Foundation

func solution(_ s:String) -> Bool
{
    var ans:Bool = false
    var p = 0
    var y = 0
    
    for i in s {
        if i == "y" || i == "Y" {
            y += 1
        } else if i == "p" || i == "P"{
            p += 1
        }
    }
    
    if p == y {
        return true
    } else {
        return false
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    pcnt = 0
    ycnt = 0
    s = s.upper()
    for a in s:
        if a == "P":
            pcnt += 1

        if a == "Y":
            ycnt += 1
    return pcnt == ycnt